packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -5,777 +5,413 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-    return account.weight;
+  function getNonvotingLockedGold() external view returns (uint256) {
+    return totalNonvoting;
   }
 
   /**
-   * @notice Returns whether or not a specified account is voting.
-   * @param account The address of the account.
-   * @return Whether or not the account is voting.
+   * @notice Returns the total amount of locked gold for an account.
+   * @param account The account.
+   * @return The total amount of locked gold for an account.
    */
-  function isVoting(address account) public view returns (bool) {
-    address voter = getDelegateFromAccountAndRole(account, DelegateRole.Voting);
-    IGovernance governance = IGovernance(registry.getAddressFor(GOVERNANCE_REGISTRY_ID));
-    IValidators validators = IValidators(registry.getAddressFor(VALIDATORS_REGISTRY_ID));
-    return (governance.isVoting(voter) || validators.isVoting(voter));
+  function getAccountTotalLockedGold(address account) public view returns (uint256) {
+    uint256 total = accounts[account].balances.nonvoting;
+    return total.add(getElection().getTotalVotesByAccount(account));
   }
 
   /**
-   * @notice Returns the weight of a commitment for a given notice period.
-   * @param value The value of the commitment.
-   * @param noticePeriod The notice period of the commitment.
-   * @return The weight of the commitment.
-   * @dev A commitment's weight is (1 + sqrt(noticePeriodDays) / 30) * value.
+   * @notice Returns the total amount of non-voting locked gold for an account.
+   * @param account The account.
+   * @return The total amount of non-voting locked gold for an account.
    */
-  function getCommitmentWeight(uint256 value, uint256 noticePeriod) public pure returns (uint256) {
-    uint256 precision = 10000;
-    uint256 noticeDays = noticePeriod.div(1 days);
-    uint256 preciseMultiplier = sqrt(noticeDays).mul(precision).div(30).add(precision);
-    return preciseMultiplier.mul(value).div(precision);
+  function getAccountNonvotingLockedGold(address account) external view returns (uint256) {
+    return accounts[account].balances.nonvoting;
   }
 
   /**
-   * @notice Returns the delegate for a specified account and role.
-   * @param account The address of the account.
-   * @param role The role to query for.
-   * @return The rewards recipient for the account.
+   * @notice Returns the account associated with `accountOrValidator`.
+   * @param accountOrValidator The address of the account or authorized validator.
+   * @dev Fails if the `accountOrValidator` is not an account or authorized validator.
+   * @return The associated account.
    */
-  function getDelegateFromAccountAndRole(
-    address account,
-    DelegateRole role
-  )
-    public
-    view
-    returns (address)
-  {
-    address delegate = accounts[account].delegates[uint256(role)];
-    if (delegate == address(0)) {
-      return account;
+  function getAccountFromValidator(address accountOrValidator) public view returns (address) {
```

Comment:
Style nit: might want to reorder a bunch of these view functions. There was a `getAccountFrom...` function earlier, might be nice to bunch "similar" functions together.

Reply:
Agree but worry it could cause merge conflicts with asaj/pos-2. How would you feel about doing this in a follow-on PR?

Reply:
Sounds good.